source/pug/views/home.pug

```diff
@@ -26,6 +29,44 @@ block content
         h3.h4-light-black Diverse Members, One Mission
         p.body-black Meet and connect with Network members.
         a.btn.btn-ghost(href=`${env.virtualRoot}people`) Member Spotlight
+    .row
+      .col
+        .mb-5.slanty-background
+          h4.h2-headings-white News & Opinions
+    -
+      data.news = data.news.filter((item) => {
+        return item.featured;
+      });
+    .row.mb-3
+      .col-md-6.mb-3.pb-5
+        //- Hardcoding this next bit because the api doesn't have a thumbnail key yet, and this will always be a video (at least for now)
```

Comment:
Can you file a follow-on ticket, if there isn't one already, to un-hardcode this?